kartothek/io/dask/bag.py

```diff
@@ -151,6 +153,14 @@ def read_dataset_as_dataframe_bag(
     dask.bag
         A dask.bag which contains the metapartitions and mapped to a function for retrieving the data.
     """
+    if isinstance(columns, dict) or isinstance(categoricals, dict):
+        warnings.warn(
+            "Trying to read a dataset with multiple internal tables. This functionality will be removed in the next "
+            "major release. If you require a multi tabled data format, we recommend to switch to the kartothek Cube "
+            "functionality https://kartothek.readthedocs.io/en/stable/index.html",
```

Comment:
You should point to the appropriate section. If we just point to the docs in general, people will get lost.

Reply:
Sure I will provide the appropriate section link.